web/src/styles/global.css

```
@@ -1,5 +1,7 @@
 @import "tailwindcss";
 @import "tw-animate-css";
+@source "../src";
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)